examples/all-options.xml

```diff
@@ -242,8 +242,8 @@
         <!-- specify what metric the selector should use to determine the optimal traversal -->
         <!-- possible values: fastetAbsolute, fastestMean, fastestMedian -->
         <selectorStrategy>fastestAbsolute</selectorStrategy>
-        <!-- set the auto tuning algorithm -->
-        <tuningStrategy>fullSearch</tuningStrategy>
+        <!-- set the tuning auto tuning pipeline -->
+        <tuningStrategies>fullSearch</tuningStrategies>
```

Comment:
Similar to my previous comment - should this be empty?

Reply:
same 4a1f9c7a8